Make multiple lines non-normative. (#3126)
modelica/ModelicaSpecification: chapters/packages.tex

It should contain the original string from the Modelica text representation.
Since in Modelica control sequences also start with a backslash and another backslash is used to use sequences literally or to hide double quotes, no change is required here.
But Modelica allows strings to go over more than one line, gettext does not.
Therefore, line breaks in Modelica must be encoded with \lstinline!"\n"!.

\begin{nonnormative}

> In order to avoid long lines Modelica strings may be split into consecutive strings at the line breaks, and the line breaks encoded with a \lstinline!"\n"! at the end of each string, e.g.

Reviewer: We can drop the part _at the line breaks_, and just rely on the example to show that this is one possible way of breaking up the lines:
```suggestion
In order to avoid long lines, strings may be split into smaller parts given on consecutive lines, e.g.
```
